lightly/api/api_workflow_client.py

```diff
@@ -67,6 +72,13 @@ def __init__(self, token: str, dataset_id: str = None, embedding_id: str = None)
         self.scores_api = ScoresApi(api_client=api_client)
         self.samples_api = SamplesApi(api_client=api_client)
 
+    def check_version_compatibility(self):
+        minimum_version = get_minimum_compatible_version()
+        if version_compare(__version__, minimum_version) < 0:
```

Comment:
We also need some environment variable here (e.g. `LIGHTLY_FORCE_CONNECTION`) as described in the issue. This should disable the exception.

Reply:
let's make another issue out of it and push it with the next update.